### ISSUE TITLE
can_parse fails unless given a base url

### ISSUE THREAD

**Issue by ethanresnick** [TkTech/can_ada#9]
From what I can gather, the second argument in `can_parse` is intended to be optional but, if it's omitted or passed explicitly as `None`, an error is thrown (from what seems to be a mistake in the bindings):

```py
can_ada.can_parse("http://2.com/", None)

Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: can_parse(): incompatible function arguments. The following argument types are supported:
    1. (input: str, base_input: str = None) -> bool

Invoked with: 'http://2.com/', None
```

**Comment by ethanresnick** [comment]
Btw, thank your for making this binding! It's very convenient :)

**Comment by TkTech** [comment]
Weirdly this seems like a pybind11 bug, where the cast to nullptr and and explicitly allowing `None` just isn't getting picked up:

```
    m.def("can_parse",
          &ada::can_parse,
          py::arg("input"),
          py::arg("base_input").none(true) = static_cast<const std::string_view*>(nullptr));
```

I'll give them a poke, as I believe this _should_ work just fine.

**Comment by TkTech** [comment]
Resolved with 1.3.0
